workspaces/global-header/plugins/global-header/src/components/SearchComponent/SearchBar.tsx

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  SearchResultState,
+  SearchResultProps,
+} from '@backstage/plugin-search-react';
+import Typography from '@mui/material/Typography';
+import { Link } from '@backstage/core-components';
+import ListItem from '@mui/material/ListItem';
+import Box from '@mui/material/Box';
+import Divider from '@mui/material/Divider';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+import Autocomplete from '@mui/material/Autocomplete';
+import TextField from '@mui/material/TextField';
+import InputAdornment from '@mui/material/InputAdornment';
+import SearchIcon from '@mui/icons-material/Search';
+import { highlightMatch } from '../../utils/stringUtils';
+import styles from './SearchBar.module.css';
```

Comment:
This will work also with dynamic plugins? I'm fine to give it a try. :)

Reply:
Yeah, this is a much cleaner solution as long as it works in showcase.. if it doesn't we can add inline styles to override the Mui default styles.